plugins/module_utils/module_container/base.py

```diff
@@ -935,6 +935,7 @@ def _compare_platform(option, param_value, container_value):
     OptionGroup(preprocess=_preprocess_healthcheck)
     .add_option('healthcheck', type='dict', ansible_suboptions=dict(
         test=dict(type='raw'),
+        test_cli_compatible=dict(type='bool'),
```

Comment:
You don't have a default here:
```suggestion
        test_cli_compatible=dict(type='bool', default=False),
```